<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogChargerGuichet</class>
 <widget class="QDialog" name="DialogChargerGuichet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Charger les couches de mon groupe</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>670</y>
     <width>201</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelMonGuichet">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Mon guichet</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFondsGeoportail">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Fonds Géoportail</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetMonGuichet">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>731</width>
     <height>191</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
   <property name="autoScroll">
    <bool>true</bool>
   </property>
   <property name="horizontalScrollMode">
    <enum>QAbstractItemView::ScrollPerItem</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::NoPen</enum>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>225</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>30</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>37</number>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>37</number>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QTableWidget" name="tableWidgetFondsGeoportail">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>731</width>
     <height>181</height>
    </rect>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
   <property name="verticalScrollMode">
    <enum>QAbstractItemView::ScrollPerItem</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::NoPen</enum>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>225</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>30</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QTableWidget" name="tableWidgetFondsGeoportailBis">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>540</y>
     <width>731</width>
     <height>121</height>
    </rect>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::NoPen</enum>
   </property>
   <property name="columnCount">
    <number>1</number>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <column/>
  </widget>
  <widget class="QLabel" name="labelFondsGeopportailBis">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>731</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Les fonds Géoportail suivants sont également paramétrés dans la carte de votre groupe mais la clé Géoportail que vous utilisez sous QGIS ne vous y donne pas accès. Pour pouvoir les charger, veuillez modifier votre clé Géoportail en vous adressant à &lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;contact.geoservices@ign.fr&lt;/span&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelGroupeActif">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
